### PR TITLE
Add more customizable keybindings for gallery image nav

### PR DESCRIPTION
**Pull request by rivertam**
Very clean codebase, by the way. I appreciate that.

### CODE REVIEW

File: src/config/Config.coffee [hunk 1]
```diff
@@ -950,7 +950,19 @@ Config =
...
+    ],
+    'Next Gallery Image': [
+      'Right',
+      'Go to the next image in gallery mode'
+    ],
+    'Previous Gallery Image': [
+      'Left',
+      'Go to the previous image in gallery mode'
+    ],
+    'Advance Gallery': [
```
Comment (rivertam) [inline src/config/Config.coffee]: Didn't know what this means, so I have no problem changing the description to actually describe it.

**Comment by rivertam** [comment]
@ccd0 I've been using this very regularly since I made this PR and it hasn't done anything bad yet. I see you're still actively developing. Any interest in this PR or should I just close it and delete my fork? I try to keep my GitHub tidy.

**Comment by ccd0** [comment]
Keep it open, I just haven't haven't had much time to work on this. The recent updates were just what was needed to keep the script working with changes on /pol/.

As for the "advance" thing, it's the same as next except in the case of videos when autoplay is turned off. In that case, one Enter seeks to the video, the next starts it. Not sure what a good concise description would be.

**Comment by rivertam** [comment]
Gotcha. Thanks. Keep up the good work. =)

**Comment by ccd0** [comment]
Looks good, merged and updated the 'Advance Gallery' description: c731f3874edb2158358d9ac4329a816ef2593fc8

**Pull request merged**